src/styles/vars.css

```diff
@@ -1,14 +1,21 @@
 :root {
   --black: #020402;
+  --dark-gray: #696969;
+  --gray: #808080;
+  --white: #ffffff;
+  --primary-axis-color: #2f4f4f;
+  --secondary-axis-color: var(--gray);
```

Comment:
A bit weird to use semantic names here, for these two variables. It would be good to store `#2f4f4f` in a colour variable, like `--dark-slate-gray` and then maybe move these two axis variables to `AxisSystem.module.css`, inside the `.axisSystem` declaration block.

Reply:
I am looking this from the extension perspective: it would make more sense to me to set variables with semantic names. 

But I see your point.

Reply:
Even `primary-dark` and `secondary-light` are not semantic names in a way, since you'd have to override them with inverted colours and the names would no longer be "valid". With fully semantic names, we'd have to use names specific to each component, like `--axis-primary-color`. Having such variables at the root would not make much sense and moving them to their respective components would make them more difficult to override... You'd also have no clue what the underlying color is without refering back to the variable's value or how it relates to another colour (e.g. whether the foreground colour darker or lighter than the background colour).

What's good with kind-of-but-not-quite-semantic colour names is that they don't completely hide what the actual colour is (or at least not the darkness/brightness aspect). This is actually a good thing when overriding with inverted colors, since you can easily double check that the relations between colours remain logical (e.g. `--primary-dark` is lighter than `--primary`).

Not sure if my reasoning is clear?

Reply:
It is. This is why I said that I saw your point :smile: